src/components/hub/elements/MessageInput.js

```diff
@@ -91,7 +92,9 @@ export default function MessageInput({
           isDisabled={true}
           isMobile={isMobile}
           placeholder={
-            "The user you are currently chatting with has deleted their account. You can no longer continue the chat."
+            !isBlockedConv
+              ? "Your opponent is offline."
```

Comment:
This message text does not help much to understand what's the reason. Should be changed later to better message text